### PR TITLE
Update metrics docs

### PR DESCRIPTION
**Pull request by gracedo**
## Description
https://jira.mesosphere.com/browse/DCOS-42214
Updates the metrics docs with the user-editable telegraf config dir that can now be used to add settings to telegraf.

## Urgency
- [ ] Blocker 
- [ ] High
- [x] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11, 1.12).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

File: pages/1.12/metrics/architecture/index.md [hunk 1]
```diff
@@ -7,9 +7,9 @@ excerpt: How DC/OS manages metrics
 enterprise: false
 ---
 
-Metrics in DC/OS, version 1.12 and newer, are based on [Telegraf](https://github.com/dcos/telegraf). Telegraf provides an agent-based service that runs on each master and agent node in a DC/OS cluster. By default, Telegraf gathers metrics from all of the processes running on the same node, processes them, then sends the collected information to a central metrics database.
```
Comment (philipnrmn) [inline pages/1.12/metrics/architecture/index.md]: Looks like your code editor chopped the hanging space here - let's not add an unnecessary change to this line.

File: pages/1.12/metrics/architecture/index.md [hunk 2]
```diff
@@ -7,9 +7,9 @@ excerpt: How DC/OS manages metrics
 enterprise: false
 ---
 
-Metrics in DC/OS, version 1.12 and newer, are based on [Telegraf](https://github.com/dcos/telegraf). Telegraf provides an agent-based service that runs on each master and agent node in a DC/OS cluster. By default, Telegraf gathers metrics from all of the processes running on the same node, processes them, then sends the collected information to a central metrics database. 
+Metrics in DC/OS, version 1.12 and newer, are based on [Telegraf](https://github.com/dcos/telegraf). Telegraf provides an agent-based service that runs on each master and agent node in a DC/OS cluster. By default, Telegraf gathers metrics from all of the processes running on the same node, processes them, then sends the collected information to a central metrics database.
 
-Telegraf has a plugin-driven architecture. The plugin architecture enables Telegraf to collect information from any supported input plugin and write results to any supported output plugin. The plugins are compiled into the Telegraf binary for execution, and you can selectively enable and customize plugins using configuration file options.
```
Comment (philipnrmn) [inline pages/1.12/metrics/architecture/index.md]: ditto, and one more at the end of the file

**Comment by philipnrmn** [comment]
Awesome. Thanks for taking care of the docs changes!

**Comment by ck4adventure** [comment]
@gracedo these are good from engg side? I'll get them put up

**Comment by gracedo** [comment]
@ck4adventure all good on our end, thanks!
